tests/unit/store/unload-test.js

```diff
@@ -86,7 +86,7 @@ test("unload a record", function(assert) {
         store.unloadRecord(record);
       });
 
-      assert.equal(get(record, 'hasDirtyAttributes'), false, "record is not dirty");
```

Comment:
I just checked and it seems that `hasDirtyAttributes` is `false` after a record is destroyed. See [this ember-twiddle](https://ember-twiddle.com/255f0c49b1f4d33bc5b9221317bdaf9c?fileTreeShown=false&numColumns=2&openFiles=routes.application.js%2Ctemplates.application.hbs). Am I missing something?

I think in this case it would be ok to add back the assertion for `hasDirtyAttributes`...